modules/partition-replica/src/main/java/org/apache/ignite/internal/partition/replica/network/TimedBinaryRow.java

```diff
@@ -35,7 +35,7 @@ public TimedBinaryRow(@Nullable BinaryRow binaryRow, @Nullable HybridTimestamp c
         this.commitTimestamp = commitTimestamp;
     }
 
-    TimedBinaryRow(@Nullable BinaryRow binaryRow) {
+    public TimedBinaryRow(@Nullable BinaryRow binaryRow) {
```

Comment:
Why it's public?

Reply:
Because now it is created in the table module, but placed in the partition-replicator (because it is needed for network messages).